src/pyhf/pdf.py

```diff
@@ -350,7 +363,9 @@ def _modifications(self, pars):
 
     def _expected_data(self, pars):
         """
-        For a single channel single sample, we compute
+        Compute the expected rates for given values of parameters.
+
+        For a single channel single sample, we compute:
 
             Pois(d | fac(pars) * (delta(pars) + nom) ) * Gaus(a | pars[is_gaus], sigmas) * Pois(a * cfac | pars[is_poi] * cfac)
```

Comment:
can we change this to the mathematical formula using the notation we introduced in the docs?

Reply:
we can do this, but I'd reserve this for another PR, this is mostly about the CI testing about docstring existence